exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_exporter.h

```diff
@@ -68,6 +68,9 @@ struct OtlpHttpExporterOptions
 
   // TODO: Enable/disable to verify SSL certificate
   std::chrono::milliseconds timeout = std::chrono::milliseconds(30000);
+
+  // filepath of the sll certifcate, this is empty if there is none. 
+  std::string sslCertPath;
```

Comment:
mTLS support (requested by someone else here in another issue) will require two more fields for paths to private and public key. To avoid confusion please rename this field to make it more clear that this is path to trusted root CA certificates, e.g. `sslCaCertPath`.
In my issue I also asked about way to specify minimum TLS version (`CURLOPT_SSLVERSION`) and allowed ciphers (`CURLOPT_SSL_CIPHER_LIST`). Below I also mentioned another one to disable certificate validation. All of them would need corresponding fields here.
BTW, you also have typo in comment above - sll -> ssl